components/Sections/ImageTextColumns/ImageTextColumns.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import prisma, { Prisma } from "@/lib/prisma";
+import { ImageTextColumnsImageBuilder } from "./ImageTextColumnsImage";
+
+export type ImageTextColumnsProps = Prisma.ImageTextColumnsGetPayload<{}> & {
+  leftImageComponent: React.ReactNode;
+  rightImageComponent: React.ReactNode;
+};
+
+export async function ImageTextColumns({
+  title,
+  content,
+  leftImage,
+  rightImage,
```

Comment:
What are `leftImage` and `rightImage` used for?
Hint: Unused variables are grayed in the editor.